packages/react/src/wishlists/hooks/types/useWishlist.ts

```diff
@@ -5,6 +5,7 @@ import type { WishlistItemHydrated } from '@farfetch/blackout-redux/entities/typ
 export type UseWishlist = () => {
   error: State['error'] | undefined;
   fetchWishlist: () => Promise<Wishlist | undefined>;
+  addWishlistItem: () => Promise<Wishlist | undefined>;
```

Comment:
I believe you're missing the payload type here (it has the same type as the respective action, with the `data` and `config`)

And since you are here, is it out of hand to double-check if [this](https://github.com/Farfetch/blackout/blob/next/packages/react/src/wishlists/hooks/types/useWishlistSets.ts#L13) has the same problem? At least for the `config` part 😬

Reply:
I had the same doubt about `config` but we don't seem to type it in any other action (they all receive `config` as a parameter). Should we add `config` to all action types? 🤔

Reply:
True, in fact, we don't type it anywhere 😅 

Should we open an issue to tackle this in all hooks?

Reply:
Good catch! I added both `data` and `config` to the action's arguments. Not having the `config` anywhere is indeed a problem, thanks for raising awareness for that problem. 

We're missing it in the hooks and in the action presets, I think we should discuss this in order to get a viable solution and then open an issue, what do you think? 😃

Reply:
Yup, we'll open an issue after a brief discussion 😄